topics/examples.md

```diff
@@ -1,61 +1,68 @@
+* [Netty](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/netty)
+* [Jetty Embedded](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/jetty-embedded)
+* [Jetty](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/jetty)
+* [Jetty WAR](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/jetty-war)
+* [Tomcat WAR](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/tomcat-war)
+* [Docker](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/docker)
+* [Google App Engine Standard](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/google-appengine-standard)
+
+### Upload and download {id="upload-download"}
+* [Upload](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/upload-file)
+* [Download](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/download-file)
+
+### Working with views {id="views"}
+* [Html](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/html)
+* [Html widget](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/html-widget)
+* [FreeMarker](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/freemarker)
+* [Velocity](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/velocity)
+* [Mustache](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/mustache)
+* [Thymeleaf](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/thymeleaf)
+
+### Serialization {id="serialization"}
+* [GSON](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/gson)
+* [Jackson](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/jackson)
+
+### Authentication and authorization {id="authentication"}
+* [Basic](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-basic)
+* [Basic with UserHashedTableAuth](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-basic-hash-table)
+* [Form](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-form)
+* [Form with HTML DSL](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-form-html-dsl)
+* [Digest](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-digest)
+* [JWT](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-jwt)
+* [JWK](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-jwk)
+* [LDAP](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-ldap)
+* [OAuth Google](https://github.com/ktorio/ktor-documentation/tree/main/codeSnippets/snippets/auth-oauth-google)
+
+### Features {id="features"}
```

Comment:
Plugins? Albeit not sure if this "category" makes sense as such since the other ones are also about plugins...

Reply:
I'd change it to Miscellaneous